source/_integrations/bmp280.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Bosch BMP280 Environmental Sensor
+description: Instructions on how to integrate a BMP280 sensor into Home Assistant.
+ha_release: 0.105
+logo: raspberry-pi.png
+ha_category:
+  - DIY
+ha_iot_class: Local Polling
+ha_quality_scale: silver
```

Comment:
@frenck can you check this?

Reply:
@klaasnicolaas We don't have to, the quality scale is now part of the integration manifest in the codebase.